doc/programs.md

```diff
@@ -164,45 +164,45 @@
 - PT, pager test.
 - PTY, pseudo-tty.
 - PWMAIL, a limited version of MAIL when not yet logged in under PWORD.
-- PWORD, replacement for sys;atsign hactrn that requires registered logins.
+- PWORD, replacement for ATSIGN HACTRN that requires registered logins.
 - QB, the game of Qubic.
 - QUOTE, prints out a random quote.
 - REATTA, reattaches disowned jobs to terminal.
 - REDRCT, redirect IP routing.
 - RIPDEV, replacement for MLDEV for no-longer-existing machines.
-- RMAIL, Mail reading client.
+- RMAIL, mail reading client.
 - RMTDEV, MLDEV for non-ITS hosts.
 - SALV, old file system tool for KA and KL.
-- SCANDL, TTY OUTPUT SPY.
+- SCANDL, TTY output spy.
 - SCHEME, Scheme interpreter.
 - SCNV, convert plotter files to XGP scan files.
-- SCRAM, encypt/decrypt file.
+- SCRAM, encrypt/decrypt file.
 - SCRMBL/UNSCR, scramble/unscramble file.
 - SEND, REPLY, replacements for DDT :SEND.
 - SENDS, Chaosnet SEND server.
-- SENSOR, sends censor.
+- SENSOR, SEND's censor.
```

Comment:
I think "sends" is a plural form here.

Reply:
I was thinking of it as censoring SEND, but yes, I guess it works both ways!

How about "An alternate message receiver", which is how it's described in DIR?

Reply:
Yes, that is more descriptive.